llvm/test/CodeGen/RISCV/rvv/masked-load-int-e64.ll

```diff
@@ -0,0 +1,47 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=riscv32 -mattr=+v -verify-machineinstrs < %s | FileCheck %s
+; RUN: llc -mtriple=riscv64 -mattr=+v -verify-machineinstrs < %s | FileCheck %s
+
+define <vscale x 1 x i64> @masked_load_nxv1i64(ptr %a, <vscale x 1 x i1> %mask) nounwind {
```

Comment:
Are these crash tests?  Or can they be pre-commited so the change is visible?

Reply:
This file doesn't crash, but the new <vscale 1 x 1 x *> tests in the other file are crashes with the new command line.